### DISCUSSION TITLE
How to implement loadbalancing using linkerd?

### DISCUSSION THREAD

**Discussion by rm185496** [linkerd/linkerd2#8832] [Help]
Hi Everyone,
I recently discovered about linkerd and was trying to implement it on one of my aks cluster.
I already have few services deployed in a namespace and multiple pods of these services are loadbalanced using K8s service(Type=LoadBalancer, This uses azure loadbalancer)
I have injected linkerd in the same namespace and I am trying to achieve loadbalancing using linkerd.
Upon using "linkerd viz dashboard &" command I am able to see cluster metrics on my localhost:50750.
Can anyone help me with implementing loadbalancing using linkerd?

**Comment by mateiidavid** [comment]
@rm185496 hey,

For __in-cluster traffic__, Linkerd will automatically loadbalance any pods that are injected (and thus part of the "mesh"). Have a look at our [docs on loadbalancing](https://linkerd.io/2.11/features/load-balancing/).
